rust/oak_log/src/lib.rs

```diff
@@ -53,8 +53,12 @@ impl Log for OakChannelLogger {
 }
 
 pub fn init(level: Level) -> Result<(), SetLoggerError> {
+    init_with(level, "log")
+}
+
+pub fn init_with(level: Level, channel_name: &str) -> Result<(), SetLoggerError> {
```

Comment:
Should this take a handle instead of a name, for consistency with the event loop method?